pkgs/by-name/ro/rose-pine-cursor/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
```

I think the finalAttrs pattern should be used here to facilitate more effective overriding. This means passing a function to `mkDerivation` instead of a set, see: https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes
```suggestion
stdenv.mkDerivation (finalAttrs: {
```